tests/runner.py

```diff
@@ -358,6 +358,11 @@ def decorator(func):
   return decorator
 
 
+def run_js_default(filename, *args, **kw):
```

Comment:
this name seems a little odd, it's not obvious what "default" means I think. how about just `run_js`? (differentiated from `jsrun.run_js` by the module it is on) that's shorter and clearer I think, and it would also make this PR's diff smaller.

Reply:
Its runs with the default engine, not the one you pass it.   This is the behaviour of the old shared.run_js wrapper that I'm deleting.

I'm trying to deliberately give it a different name so i can easily grep for the different types of run_js users.. just like I did with run_js_tool which is used for compiler tools.

Part of the reason for this change is to followup with another refactor so I expect all these lines to change again soon anyway.

Reply:
that might be useful for grepping temporarily, but do you agree it's less readable and worth removing eventually? it just feels non-pythonic to have "default" in the name instead of `, engine=..` as a default param.

Reply:
Is there a better name you can think of?  I want to make it a distinct function to it can't collide with the one in the jsrun module.

How about "run_output_js"?  Or "run_test_output"?

Reply:
I agree its a bad name.. I promise to fix it in the next couple of PRs.

Reply:
Is `run_js` not an ok name? I think it's ok to have the same name in different modules. The module differentiates them. (Or if there is no difference, then we need just one?)